### DISCUSSION TITLE
Configurable port number ?

### DISCUSSION THREAD

**Discussion by philvil** [dorssel/usbipd-win#725] [Q&A]
Hi everyone,
I try using usbipd-win with a docker developpement container running Debian 11. Unfortunately it seems that Docker doesn't like the use of 3240 port for some reasons. It has been announced that the version 3.2.0  of usbipd-win offers configurable port number. How can I use this feature ?
Thanks in advance and best regards

**Comment by dorssel** [comment]
See https://github.com/dorssel/usbipd-win/issues/679.
